pkgs/by-name/tl/tlock/package.nix

```diff
@@ -0,0 +1,41 @@
+{
+  lib,
+  fetchFromGitHub,
+  buildGoModule,
+}:
+
+buildGoModule rec {
+  pname = "tlock";
+  version = "1.0.0";
+
+  src = fetchFromGitHub {
+    owner = "eklairs";
+    repo = "tlock";
+    rev = "v${version}";
+    hash = "sha256-O6erxzanSO5BjMnSSmn89w9SA+xyHhp0SSDkCk5hp8Y=";
+    fetchSubmodules = true;
+  };
+
+  vendorHash = "sha256-G402CigSvloF/SI9Wbcts/So1impMUH5kroxDD/KKew=";
+
+  excludedPackages = [
+    "bubbletea"
+    "scripts"
+  ];
+
+  ldflags = [
+    "-s"
+    "-w"
+    "-X github.com/eklairs/tlock/tlock-internal/constants.VERSION=v${version}"
+  ];
```

Comment:
```suggestion
```
Are these really necessary?

Reply:
Yes, they are.
1. `-s` and `-w` are used to reduce the size of the binary
2. `"-X github.com/eklairs/tlock/tlock-internal/constants.VERSION=v${version}"` is used to override the value of a constant during time with the version of the package (which is then used internally to show the package version)